pkg/vault/contracts/ProtocolFeeController.sol

```diff
@@ -231,8 +233,11 @@ contract ProtocolFeeController is
                 }
 
                 if (needToSplitFees) {
-                    uint256 totalVolume = feeAmounts[i].divUp(aggregateFeePercentage);
-                    uint256 protocolPortion = totalVolume.mulUp(protocolFeePercentage);
+                    // The Vault took a single "cut" for the aggregate total percentage (protocol + pool creator) for
+                    // this fee type (swap or yield). Now we need to "disaggregate" the total fee amount and divide
+                    // it between the protocol and pool creator, according to their individual percentages.
+                    uint256 totalAggregateFeeCollected = feeAmounts[i].divUp(aggregateFeePercentage);
```

Comment:
Let's call it the same as in the Vault: https://github.com/balancer/balancer-v3-monorepo/blob/main/pkg/vault/contracts/Vault.sol#L1046 `totalFeeAmount`.

Let's also clarify that the first step is actually reconstructing the total fees, which are used to compute the protocol fees in the first place, and the remainder goes to the pool creator.

I think this is more or less clear elsewhere, but the nuances around this are kinda easy to forget, so the more comments we add, the better.